### ISSUE TITLE
(Suggestion) Tapping and moving stylus on touchscreen click and drag

### ISSUE THREAD

**Issue by PolarisBeaver** [TaraHoleInIt/minivmac-3ds#1]
This should be an option, especially for doing drag and drop events, or just quickly tapping something to select it

**Comment by TaraHoleInIt** [comment]
So like, acting like the mouse button is always down when the touch screen receives input?
I'll have to think about how I'd implement that or make it toggleable.

**Comment by PolarisBeaver** [comment]
Yup, just like that. It would be nice, as a toggleable option, so if someone does not wish to have this, they are not forced to.
